[2025-07-12T13:43:21.746+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-12T13:43:21.766+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: employee_ingestion_dag.split_and_save_data manual__2025-07-12T13:43:17+00:00 [queued]>
[2025-07-12T13:43:21.776+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: employee_ingestion_dag.split_and_save_data manual__2025-07-12T13:43:17+00:00 [queued]>
[2025-07-12T13:43:21.779+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-12T13:43:21.793+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): split_and_save_data> on 2025-07-12 13:43:17+00:00
[2025-07-12T13:43:21.801+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1159) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-12T13:43:21.802+0000] {standard_task_runner.py:72} INFO - Started process 1163 to run task
[2025-07-12T13:43:21.802+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'employee_ingestion_dag', 'split_and_save_data', 'manual__2025-07-12T13:43:17+00:00', '--job-id', '239', '--raw', '--subdir', 'DAGS_FOLDER/ingestion_pipeline.py', '--cfg-path', '/tmp/tmpvfae1y0h']
[2025-07-12T13:43:21.804+0000] {standard_task_runner.py:105} INFO - Job 239: Subtask split_and_save_data
[2025-07-12T13:43:21.854+0000] {task_command.py:467} INFO - Running <TaskInstance: employee_ingestion_dag.split_and_save_data manual__2025-07-12T13:43:17+00:00 [running]> on host f80ef267dd5d
[2025-07-12T13:43:21.939+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='employee_ingestion_dag' AIRFLOW_CTX_TASK_ID='split_and_save_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-12T13:43:17+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-12T13:43:17+00:00'
[2025-07-12T13:43:21.940+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-12T13:43:22.364+0000] {logging_mixin.py:190} INFO - Processing file: employee_data2_with_errors_chunk_13.csv
[2025-07-12T13:43:22.365+0000] {logging_mixin.py:190} INFO - Total rows: 2955, Invalid rows: 566
[2025-07-12T13:43:22.389+0000] {logging_mixin.py:190} INFO - File split - Good rows: /opt/airflow/data/good_data/employee_data2_with_errors_chunk_13.csv, Bad rows: /opt/airflow/data/bad_data/employee_data2_with_errors_chunk_13.csv
[2025-07-12T13:43:22.391+0000] {logging_mixin.py:190} INFO - Original file removed: /opt/airflow/data/raw_data/employee_data2_with_errors_chunk_13.csv
[2025-07-12T13:43:22.393+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-12T13:43:22.404+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-12T13:43:22.405+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=employee_ingestion_dag, task_id=split_and_save_data, run_id=manual__2025-07-12T13:43:17+00:00, execution_date=20250712T134317, start_date=20250712T134321, end_date=20250712T134322
[2025-07-12T13:43:22.458+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-12T13:43:22.478+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-12T13:43:22.480+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
